### ISSUE TITLE
Surprised to see that they're giving us the raw SVG

### ISSUE THREAD

**Issue by subhashisbhowmik** [janghaludu/cowin-captcha#2]
Surprised to see that they're giving us the raw SVG, should be able to read the path for easier parsing

**Comment by janghaludu** [comment]
You're right!!! Just noticed that there are always 5 path objects in every file with 5 characters even if the characters are intersecting!

**Comment by subhashisbhowmik** [comment]
Yes, and easily distinguishable from the small spline paths (strokes vs fills)

**Comment by subhashisbhowmik** [comment]
![Example](https://user-images.githubusercontent.com/12123763/119664380-e72bca80-be50-11eb-9fab-3781d3e27c7a.png)

**Comment by janghaludu** [comment]
Neato. Implementing this will definitely give a boost to accuracy. Will try doing this and replacing Zernike Polynomials with  HOG descriptors over the weekend if I get time. Feel free to give it a go :)

**Comment by janghaludu** [comment]
Managed to get probably 100% accuracy with average time of 500 milliseconds now :) I'm yet to find a CAPTCHA that is decoded wrongly in 500 consecutive tests (2500 characters). Appreciate the tip. Will update the repo soon.
